tv_apps/browser/js/index.js

```diff
@@ -961,7 +961,7 @@ var Browser = {
           case 'url':
             var url = this.getUrlFromInput(activity.source.data.url);
             this.debug(' url = ' + url);
-            this.start_page_url = url;
+            this.navigate(url);
```

Comment:
Please refer to the ```openPage``` method in  ```js/connection_handler.js```. We open a new tab when the current browser tab is not empty.